src/flexmeasures_client/client.py

```diff
@@ -247,6 +249,8 @@ async def trigger_storage_schedule(
         check_for_status(status, 200)
         print("Schedule triggered successfully.")
 
+        return response.get("schedule")
```

Comment:
Maybe we should also set the response type to str if we are returning something now?

Reply:
yeah! I was going to add it but then I forgot haha